spring-cloud-starter-stream-sink-cassandra/src/main/java/org/springframework/cloud/stream/app/cassandra/CassandraProperties.java

```diff
@@ -136,6 +148,10 @@ public void setMetricsEnabled(boolean metricsEnabled) {
 		this.metricsEnabled = metricsEnabled;
 	}
 
+	public void setUseSsl(boolean useSsl) { this.useSsl = this.useSsl; }
```

Comment:
Pay attention, please, how you did `this.useSsl = this.useSsl` where we expect as `this.useSsl = useSsl`.

That's why tests are always good way to be sure we don't miss similar simple mistakes.